cpp/simple_crawler.cc

```diff
@@ -67,90 +58,6 @@ void Usage() {
-    if (url_ignore_regex_matcher == nullptr)
-        ERROR("could not initialize URL regex matcher\n"
-              + err_msg);
-
-    if (url_ignore_regex_matcher->matched(url)) {
-        logger->warning("Skipping URL: " + url);
-        return;
-    }
-
-    Downloader::Params params;
-    params.user_agent_            = USER_AGENT;
-    params.acceptable_languages_  = acceptable_languages;
-    params.honour_robots_dot_txt_ = not ignore_robots_dot_txt;
-    min_url_processing_time->sleepUntilExpired();
-    Downloader downloader(url, params, timeout);
-    min_url_processing_time->restart();
-    if (downloader.anErrorOccurred()) {
-        logger->warning("Failed to retrieve a Web page (" + url + "):\n"
-                        + downloader.getLastErrorMessage());
-        return;
-    }
-
-    const std::string message_headers(downloader.getMessageHeader()), message_body(downloader.getMessageBody());
-    if (print_redirects) {
-        std::list<std::string> location_urls;
-        ExtractLocationUrls(message_headers, &location_urls);
-        for (const auto &location_url : location_urls)
-            std::cout << "Location: " << location_url << '\n';
-        std::cout << "\n\n";
-    }
-
-    if (all_headers or last_header) {
-        if (all_headers or last_header)
-            std::cout << StringUtil::ReplaceString("\r\n", "\n", message_headers) << '\n';
-    }
-
-    static const unsigned EXTRACT_URL_FLAGS(WebUtil::IGNORE_DUPLICATE_URLS | WebUtil::IGNORE_LINKS_IN_IMG_TAGS
-                                            | WebUtil::REMOVE_DOCUMENT_RELATIVE_ANCHORS
-                                            | WebUtil::CLEAN_UP_ANCHOR_TEXT
-                                            | WebUtil::KEEP_LINKS_TO_SAME_MAJOR_SITE_ONLY
-                                            | WebUtil::ATTEMPT_TO_EXTRACT_JAVASCRIPT_URLS);
-
-    std::vector<WebUtil::UrlAndAnchorTexts> urls_and_anchor_texts;
-    WebUtil::ExtractURLs(message_body, url, WebUtil::ABSOLUTE_URLS, &urls_and_anchor_texts, EXTRACT_URL_FLAGS);
-    for (const auto &url_and_anchor_texts : urls_and_anchor_texts) {
-        const std::string extracted_url(url_and_anchor_texts.getUrl());
-        if (url_regex_matcher.matched(extracted_url))
-            extracted_urls->emplace(extracted_url);
-    }
-
-    --remaining_crawl_depth;
-    if (remaining_crawl_depth > 0) {
-        for (const auto &url_and_anchor_texts : urls_and_anchor_texts)
-            ProcessURL(url_and_anchor_texts.getUrl(), all_headers, last_header, ignore_robots_dot_txt,
-                       print_redirects, timeout, min_url_processing_time, acceptable_languages, remaining_crawl_depth,
-                       url_regex_matcher, extracted_urls);
-    }
-}
-
-
 static struct option options[] = {
     { "help",                    no_argument,              nullptr, 'h'  },
```

Comment:
I suspect that there are 2 spaces before the closing curly bracket instead of only one.